src/module.ts

```diff
@@ -2,12 +2,15 @@ import { fileURLToPath } from 'node:url'
 import { defineNuxtModule, addComponentsDir, addPlugin, addServerHandler } from '@nuxt/kit'
 import { join } from 'pathe'
 import defu from 'defu'
+import fs from 'fs'
 
 export interface ModuleOptions {
   /** It is recommended you set the secret key via `runtimeConfig.turnstile.secretKey` or NUXT_TURNSTILE_SECRETKEY */
   secretKey?: string
   /** Your Turnstile site key */
   siteKey?: string
+  /** Path to the secret key */
+  secretKeyPath?: string
```

Comment:
Maybe move this directory under the `secretKey` so that the alternative is easy to see.

Reply:
This thread can be resolved :+1: